src/main/java/io/vertx/core/shareddata/AsyncMap.java

```diff
@@ -134,4 +141,84 @@
    */
   void size(Handler<AsyncResult<Integer>> resultHandler);
 
+  /**
+   * Get the keys of the map, asynchronously.
+   * <p>
+   * Use this method with care as the map may contain a huge number of entries.
```

Comment:
huge -> large, that may not fit entirely in memory.

Reply:
Done